cpp/src/Ice/ConnectionI.cpp

```diff
@@ -3535,15 +3542,7 @@ Ice::ConnectionI::scheduleTimeout(SocketOperation status)
     int timeout;
```

Comment:
It would be clear to declare this variable as:

```
chrono::milliseconds timeout;
```

Reply:
This code is temporary. I will use a chrono duration in a follow-up PR.